pkg/ccl/backupccl/backuppb/backup.proto

```diff
@@ -206,3 +206,18 @@ message ExportStats {
   int64 duration = 3 [(gogoproto.casttype) = "time.Duration"];
 }
 
+// SSTBatcherStats is a message containing information about the batches created
+// by the SST Batcher.
+message SSTBatcherStats {
```

Comment:
Noted below but what if we turn `ingestionStats` from a struct into a protobuf and emit that instead? This way we avoid two parallel tracks of collecting stats. We could process those stats to pick out the useful pieces of information that we wish to display but its easier to reason about a single source of truth.